README.md

```diff
@@ -275,13 +275,16 @@ files and start again.
 vcpkg is the easiest way to have dependencies installed. It downloads packages sources, headers and build libraries for whatever TRIPLET is set up (platform/arq).
 VCPKG maintains any installed package inside its own folder, allowing to have multiple vcpkg folder with different dependencies installed on each. This is also great because you don't have to install dependencies globally on your system.
 
-Follow next steps to install VCPKG and have it linked to cmake
+Follow next steps to install VCPKG and have it linked to cmake. The vcpkg repository is checked out at the ref in [vcpkg.yml](eng/pipelines/templates/steps/vcpkg.yml#L11). Azure SDK code in this version is known to work at that vcpkg ref.

```

Comment:
Using a tagged release makes it easier to explain what version we are on. Optionally, we may not need to include the link to the yml file, but rather state the version in the file directly.

OR, we may want to have a vcpkg version file which contains the tag/commit, and we can point folks to that.

Also, unless we use a permalink, this line number could shift as the yml file changes over time.
So, if we are keeping things as is, should we use a premalink?

```suggestion
Follow next steps to install VCPKG and have it linked to cmake. The vcpkg repository is checked out at the ref in [vcpkg.yml](https://github.com/Azure/azure-sdk-for-c/blob/ebbfb516315a5ca4681f1da284d7e0d8bee4656a/eng/pipelines/templates/steps/vcpkg.yml#L11). Azure SDK code in this version is known to work at that vcpkg ref.

```

Reply:
This link will change over time and the version in the master branch should continue to float. When we generate the links for doc generation we will generate permalinks that will refer to the SHA.

Reply:
If we have to update the link in the README over time anyway, should we just put the release tag we use directly in the text, inline, rather than having an indirection through a link to the yml file?

Reply:
Splitting this out into its own issue: https://github.com/Azure/azure-sdk-for-c/issues/928